tests/testdata/demo-modules/demo-module-2/load.php

```diff
@@ -2,7 +2,6 @@
 /**
  * Module Name: Demo Module 2
  * Description: This is the description for demo module 2.
- * Focus: something
```

Comment:
See above, this file still needs to be moved accordingly (to a `something` directory, which of course is a focus that doesn't actually exist). This relates to my other comment above, parsing modules shouldn't be coupled to which focuses we have explicitly defined.

Reply:
👍🏼